schema.sql

```diff
@@ -45,3 +47,10 @@ CREATE TABLE visits(
   date_of_visit DATE,
   PRIMARY KEY(id)
 );
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- Solution
+CREATE INDEX animal_id ON visits(animal_id ASC);
+CREATE INDEX visits_vet_id ON visits(vet_id ASC);
+CREATE INDEX email_index ON owners(email ASC);
```

Comment:
![image](https://github.com/mershark/database-performance-audit/assets/93039359/82497a87-9aec-4663-bf17-4cf71f774c38)

- [ ] It would be nice if you make the Execution time more than 1000ms before adding the solutions because you are supposed to reduce the Execution to less than 1000ms so it means that it should be more than 1000ms then you can reduce it to less than 1000ms 🤔 Also it will be good for your practice to reduce the Execution time of the Databases because in the near future when you will be working in big companies then you will be having big databases with a lot of Execution time to reduce then your skills and knowledge will help you a lot which you will learn in this and the near projects 👍